docs/demo/materials/03-atoms/tab.html

```diff
@@ -8,12 +8,15 @@
 
     `dc-tab__element`: Use this class to create another tab.
 
+    `dc-tab__element--active`: to show a tab as header.
```

Comment:
this should be `dc-tab__element--header`

Reply:
Good catch! Now, it is fixed.